platform_umbrella/apps/common_ui/assets/tailwind.config.js

```diff
@@ -70,9 +70,6 @@ module.exports = {
     //
     //     <div class="phx-click-loading:animate-ping">
     //
-    plugin(({ addVariant }) =>
-      addVariant('phx-feedback', ['label:not(.phx-no-feedback) &'])
-    ),
     plugin(({ addVariant }) =>
       addVariant('phx-no-feedback', ['.phx-no-feedback&', '.phx-no-feedback &'])
```

Comment:
Do we still need this if we aren't using the inverse?

Reply:
Yes, we still need the `phx-no-feedback:*` classes. I was just using `phx-feedback:*` in some places to simplify the long classnames, but it turns out that doesn't work very well.